### ISSUE TITLE
Revert Part

### ISSUE THREAD

**Issue by Richardson-Chong** [irapkaist/removert#21]
Thanks for your invential method. And can you apply the revert part that for the ground-disappeared weakness of only remove part.

**Comment by qweesiic** [comment]
yes, i also found nothing about revert part in the source code . could you share this part? thanks

**Comment by Richardson-Chong** [comment]
After reading your paper, I infer your intention in revert is that compare the differences in range between the dynamic points been removed ans the series of low resolution rimg. Is that right?

**Comment by TongxingJin** [comment]
> After reading your paper, I infer your intention in revert is that compare the differences in range between the dynamic points been removed ans the series of low resolution rimg. Is that right?

I do agree with you. Have you ever tried this idea?

**Comment by Richardson-Chong** [comment]
I tried in the same way as remove, and it worked. But I didn't check the precision and compared the situations with eyes instead- v -

**Comment by TongxingJin** [comment]
> I tried in the same way as remove, and it worked. But I didn't check the precision and compared the situations with eyes instead- v -

Did you compare the rimg of dynamic points with all of the scans as REMOVE ? How about the pixel resolution you set?

**Comment by gisbi-kim** [comment]
dear @Richardson-Chong and all guys,
sorry for the late response. 
The reverting part has been incorporated within ltremovert code (see https://github.com/gisbi-kim/lt-mapper/tree/main/ltremovert) 
but not committed to this repo yet.
